ob-pattern-2/templates/ob-bi/dashboard/wso2ob-pattern-2-ob-bi-dashboard-bin.yaml

```diff
@@ -298,21 +309,23 @@ data:
         -XX:HeapDumpPath="$RUNTIME_HOME/logs/heap-dump.hprof" \
         $JAVA_OPTS \
         -classpath "$CARBON_CLASSPATH" \
-        -Djava.endorsed.dirs="$JAVA_ENDORSED_DIRS" \
+        $JAVA_VER_BASED_OPTS \
         -Djava.io.tmpdir="$CARBON_HOME/tmp" \
         -Dcarbon.registry.root=/ \
         -Djava.command="$JAVACMD" \
         -Dcarbon.home="$CARBON_HOME" \
         -Dwso2.runtime.path="$RUNTIME_HOME" \
         -Dwso2.runtime="$RUNTIME" \
+        -Djavax.xml.bind.JAXBContextFactory="com.sun.xml.bind.v2.ContextFactory" \
         -Djava.util.logging.config.file="$RUNTIME_HOME/bin/bootstrap/logging.properties" \
         -Djava.security.egd=file:/dev/./urandom \
         -Dfile.encoding=UTF8 \
         -Djavax.net.ssl.keyStore="$CARBON_HOME/resources/security/wso2carbon.jks" \
         -Djavax.net.ssl.keyStorePassword="wso2carbon" \
         -Djavax.net.ssl.trustStore="$CARBON_HOME/resources/security/client-truststore.jks" \
         -Djavax.net.ssl.trustStorePassword="wso2carbon" \
+        -Dio.netty.tryReflectionSetAccessible=false \
         org.wso2.carbon.launcher.Main $*
         status=$?
     done
-    {{ end }}
+
```

Comment:
@nalakagunathilaka let's remove this additional EOF line. Please check other files for similar cases and remove them if any.

Reply:
Fixed with bfff48b.